rocAL/rocAL_pybind/examples/rocAL_api_python_unittest.py

```diff
@@ -52,154 +73,290 @@ def main():
+                                rocal_tensor_output_datatype=tensor_dtype)
+            output = fn.blend(images,
+                              output1,
+                              ratio=0.5,
+                              rocal_tensor_output_layout=tensor_layout,
+                              rocal_tensor_output_datatype=tensor_dtype)
+        elif augmentation_name == "resize_crop":
+            output = fn.resize_crop(images,
+                                    resize_width=640,
+                                    resize_height=480,
+                                    crop_area_factor=0.25,
+                                    crop_aspect_ratio=1.2,
+                                    x_drift=0.6,
+                                    y_drift=0.4,
+                                    rocal_tensor_output_layout=tensor_layout,
+                                    rocal_tensor_output_datatype=tensor_dtype)
+        elif augmentation_name == "center_crop":
+            output = fn.center_crop(images,
+                                    crop=[2, 224, 224],
+                                    rocal_tensor_output_layout=tensor_layout,
+                                    rocal_tensor_output_datatype=tensor_dtype)
+
+        if output_set == 0:
+            pipe.set_outputs(output)
     # build the pipeline
     pipe.build()
     # Dataloader
-    data_loader = ROCALClassificationIterator(pipe,device=device,device_id=local_rank)
+    data_loader = ROCALClassificationIterator(pipe, device=device, device_id=local_rank)
     cnt = 0
     import timeit
     start = timeit.default_timer()
 
     # Enumerate over the Dataloader
     for epoch in range(int(args.num_epochs)):
         print("EPOCH:::::", epoch)
-        for i, it in enumerate(data_loader, 0):
-            if args.print_tensor:
-                print("**************", i, "*******************")
-                print("**************starts*******************")
-                print("\nImages:\n", it[0])
-                print("\nLABELS:\n", it[1])
-                print("**************ends*******************")
-                print("**************", i, "*******************")
-            for img in it[0]:
-                cnt = cnt+1
-                if display:
-                    draw_patches(img, cnt, rocal_device)
-
-            break
+        for i, (output_list, labels) in enumerate(data_loader, 0):
+            for j in range(len(output_list)):
+                if args.print_tensor:
+                    print("**************", i, "*******************")
+                    print("**************starts*******************")
+                    print("\nImages:\n", output_list[j])
+                    print("\nLABELS:\n", labels)
+                    print("**************ends*******************")
+                    print("**************", i, "*******************")
+                draw_patches(output_list[j], cnt, rocal_device)
```

Comment:
@SundarRajan28 are we changing this to show the image for all iterations? it used to only for the 1st?

Reply:
We modified the python unit tests to replicate the same functionality as C++ unit tests which dumps all the iteration outputs with the same filename. The last iteration outputs is taken as the augmentation output to be compared against the golden outputs